samples/stock/demo/inverted-navigator/demo.js

```diff
@@ -0,0 +1,457 @@
+    y: 1,
+    lbl: 'Artificial intelligence-assisted coding',
+    details: ''
+}];
+
+Highcharts.stockChart('container', {
+    chart: {
+        inverted: true,
+        zooming: {
+            mouseWheel: {
+                enabled: false
+            }
+        }
+    },
+    title: {
+        text: 'List of stock historic inventions.',
+        align: 'left'
+    },
+    subtitle: {
+        text: 'Source: Wikipedia',
+        align: 'left'
+    },
+    rangeSelector: {
+        buttons: [{
+            type: 'year',
+            count: 30,
+            text: '30y'
+        }, {
+            type: 'year',
+            count: 100,
+            text: '100y'
+        }]
+    },
+    navigator: {
+        opposite: true,
+        series: {
+            type: 'line',
+            marker: {
+                enabled: true
+            },
+            lineWidth: 0
+        },
+        xAxis: {
+            ordinal: false,
+            reversed: false,
+            tickPixelInterval: 50
+        }
+    },
+    xAxis: {
+        ordinal: false,
+        labels: {
+            enabled: false
+        },
+        tickLength: 0,
+        lineWidth: 0,
+        min: Date.UTC(1900, 0, 1),
+        reversed: false
+    },
+    yAxis: {
+        min: 0.75,
+        max: 2,
+        labels: {
+            enabled: false
+        },
+        gridLineWidth: 0
+    },
+    tooltip: {
+        shared: false,
+        useHTML: true,
+        formatter: function () {
+            let tooltipString = '<div>';
+
+            if (this.series.name === 'Series 1') {
+                return false;
+            }
+
+            if (this.point.options.location) {
+                tooltipString = '<div style="width: 140px;">Location: ' +
+                    this.point.options.location + '</div>';
+            }
+            tooltipString += '<div style="width: 140px;">Year: ' +
+                Highcharts.dateFormat('%Y', this.x) + '</div>';
+            if (this.point.options.details) {
+                tooltipString += '<div style="width: 300px;"> ' +
+                    this.point.options.details + '</div>';
+            }
+            return tooltipString + '</div>';
+        }
+    },
+    series: [{
+        pointStart: Date.UTC(1600, 0, 1),
+        pointInterval: 5,
+        pointIntervalUnit: 'year',
+        data: [
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75, 0.75,
+            0.75, 0.75, 0.75, 0.75, 0.75, 0.75
+        ],
```

Comment:
This whole concept looks like a hack - it looks like we're shoehorning a column series into something that it wasn't meant to be. I suspect the original `stock/navigator/inverted` demo was created before we had the timeline series. Instead of this trick, I think we should check whether we can use a true timeline series for this demo: https://jsfiddle.net/highcharts/f8vzms2c/